### PR TITLE
OADP 2985 - Performance fine tuning of the cpu and memory for Kopia and…

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.3.0

OCP 4.13+

Resolves - https://issues.redhat.com/browse/OADP-2985

Deploy preview - https://67089--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#oadp-setting-resource-limits-and-requests_installing-oadp-aws

(The same change appears in each of the cloud providers)

### CODE REVIEW

**Review by weshayutin — APPROVED** [review]
This looks good and sufficient to me.   At a later release 1.3.1 or 1.3.2 it would be appropriate to augment this statement with a brief example from the performance team.  @m0rdechai

**Comment by CarmiWisemon** [comment]
@amastbau 
Can you please review the PR?

**Comment by amastbau** [comment]
@weshayutin  is this note specific for native data mover?
Filesystem Kopia is not tech preview, correct?

**Comment by CarmiWisemon** [comment]
@amastbau 
Kopia is not in tech preview.
This is a general note in the Installation section, unrelated to Data Mover, which will appear in 1.2 and 1.3.
It will inform the user that in 1.3 Kopia can be used as an uploader and that it is resource intensive.

**Comment by amastbau** [comment]
OK, so this is referring to filesystem..thanks!.
LGTM

File: modules/oadp-setting-resource-limits-and-requests.adoc [hunk 1]
```diff
@@ -38,4 +38,9 @@ spec:
             memory: 256Mi
 ----
 <1> Specify the node selector to be supplied to Velero podSpec.
-<2> The `resourceAllocations` listed are for average usage.
\ No newline at end of file
+<2> The `resourceAllocations` listed are for average usage.
+
+[NOTE]
+====
+In OADP version 1.3 and later, Kopia can be used as an uploader. Kopia can be resource intensive, and you might need to adjust the CPU and memory requirements accordingly.
```
Comment (weshayutin) [inline modules/oadp-setting-resource-limits-and-requests.adoc:45]: s/In OADP version 1.3 and later, Kopia can be used as an uploader/Kopia is now an option in OADP 1.3. and later releases.  Kopia is used for both file system backups and data mover use cases.   Kopia is more resource intensive than Restic, and you might need to adjust the CPU and memory requirements accordingly.

Comment (amastbau) [reply modules/oadp-setting-resource-limits-and-requests.adoc:45]: maybe clarify that with the  (native) data mover, you do not have any other option than Kopia.
@weshayutin

Comment (weshayutin) [reply modules/oadp-setting-resource-limits-and-requests.adoc:45]: Yes.. I agree. Mentioning that kopia is the only option for data mover use cases is important.  @CarmiWisemon do you have enough info to slightly rewrite?

Comment (CarmiWisemon) [reply modules/oadp-setting-resource-limits-and-requests.adoc]: @weshayutin @amastbau 
I made the update you requested.

File: modules/oadp-setting-resource-limits-and-requests.adoc [hunk 2]
```diff
@@ -38,4 +38,9 @@ spec:
             memory: 256Mi
 ----
 <1> Specify the node selector to be supplied to Velero podSpec.
-<2> The `resourceAllocations` listed are for average usage.
\ No newline at end of file
+<2> The `resourceAllocations` listed are for average usage.
+
+[NOTE]
+====
+Kopia is an option in OADP 1.3 and later releases. You can use Kopia for both file system backups and data mover use cases. You must use Kopia for the Built-in DataMover. Kopia is more resource intensive than Restic, and you might need to adjust the CPU and memory requirements accordingly.
```
Comment (amastbau) [inline modules/oadp-setting-resource-limits-and-requests.adoc:45]: You can use Kopia for file system backups, and it is the only option for data mover cases with the Built-in DataMover.

Comment (amastbau) [reply modules/oadp-setting-resource-limits-and-requests.adoc:45]: and then the rest, ofcourse

**Comment by amastbau** [comment]
LGTM

**Review by amastbau — APPROVED** [review]

**Comment by ShaunaDiaz** [comment]
@CarmiWisemon Just curious if "Built-in" as capitalized is part of the component or CR name here? Otherwise, LGTM!

**Review by ShaunaDiaz — APPROVED** [review]

**Comment by CarmiWisemon** [comment]
> @CarmiWisemon Just curious if "Built-in" as capitalized is part of the component or CR name here? Otherwise, LGTM!
@ShaunaDiaz 
It is the name of the component. Thank you for your review!

**Review by stillalearner — APPROVED** [review]
lgtm

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**